### ISSUE TITLE
Windows builds failing to load xunit file

### ISSUE THREAD

**Issue by nguerrera** [dotnet/corefx#3285]
This is reproducing in CI and locally

```
Unhandled Exception: System.IO.FileNotFoundException: 
Could not load file or assembly 'xunit.runner.utility.*dotnet*, Version=2.1.0.3168, Culture=neutral, 
PublicKeyToken=8d05b1bb7a6fdb6c' or one of its dependencies. The system cannot fi
  nd the file specified.
     at Xunit.ConsoleClient.Program.Main(String[] args)
```

The test directory does not have this file, it has only:

```
 Directory of D:\Src\corefx\bin\tests\Windows_NT.AnyCPU.Debug\System.IO.Tests\dnxcore50

09/16/2015  05:04 PM           111,104 xunit.runner.utility.desktop.dll
09/16/2015  05:04 PM            67,584 xunit.runner.utility.dnx.dll
```

However, xunit.console.netcore.exe there references it.

It's a mystery how this could have gotten through CI. Did a package change in place on myget or something?

cc @weshaggard @mellinoe @ianhays @sokket @stephentoub

**Comment by ianhays** [comment]
I recently updated buildtools to use xunit rc1 which follows this change in naming for the xunit runner, but that is in a new buildtools package on myget (00085)

**Comment by ianhays** [comment]
This is the PR: https://github.com/dotnet/buildtools/pull/265

**Comment by mellinoe** [comment]
Hm, for whatever reason the build is pulling in xunit.console.netcore v1.0.2-prerelease-00088 (the latest one).

The project.lock.json file for our test-runtime lists the dependency at version 00085, but after the build this changes to 00088 (even though it is locked). I guess it's possible that whoever modified the project.json file last forgot to lock it at the same time, so now all of the dependencies are floating up to the latest version.

We need to probably publish a new version of buildtools that has this dependency properly locked down, or explicitly specified. That would bring us back to normal, I think.

As an aside, I personally think we should do away with all of our usage of wildcard versioning, and probably stop checking in lock files as well. I don't really think it buys us anything at all. But that's a separate discussion.

**Comment by weshaggard** [comment]
I suspect what this is likely a combination of things. One was the move to DNU beta7 earlier today which causes the lock files to be invalidated. The second issue is that @ianhays published some new packages today which have these dependencies which is causing the latest versions to be picked up. This can be addressed by updating buildtools lock file again. I'm looking into it.

**Comment by weshaggard** [comment]
PR https://github.com/dotnet/corefx/pull/3286 should fix this issue.

**Comment by weshaggard** [comment]
Fixed by #3286
